streams/es.m3u

```diff
@@ -71,8 +67,6 @@ https://directes-tv-int.ccma.cat/live-origin/bondia-hls/master.m3u8
 https://www.burrianateve.com/hls/btv.m3u8
 #EXTINF:-1 tvg-id="CadenaElite.es",Cadena Elite (720p) [Not 24/7]
 https://cloudvideo.servers10.com:8081/8004/index.m3u8
-#EXTINF:-1 tvg-id="Calle13Universal.es",Calle 13 Universal (1080p)
-https://spa-ha-p004.cdn.masmediatv.es/SVoriginOperatorEdge3/smil:96_HD.smil/manifest.m3u8
```

Comment:
https://spa-ha-p003.cdn.masmediatv.es

Reply:
Oh, thank you!
I guess I will have to replace every spa-ha-p004.cdn.masmediatv.es by this domain, but thanks!

Reply:
Not all links work, but that may gives you a path to begin with.